zhaquirks/const.py

```diff
@@ -89,6 +96,7 @@
 OUTPUT_CLUSTERS = SIG_EP_OUTPUT
 PARAMS = "params"
 PRESS_TYPE = "press_type"
+PRESSED = "initiali_switch_press"
```

Comment:
initiali?

Reply:
Way not **half** _Italiano_ WiTh CaMeLe CaSe ?
Fixed.

Reply:
Is OK now ?

Reply:
Yes, ILGTM